README.md

```diff
@@ -0,0 +1,143 @@
+    - Smart contracts that can be considered a part of the Maker Core Protocol, and are immutable and present in all Allocators.
+    - Their main role is to mint NST (New Stable Token) and hold it (possibly with other tokens) in the AllocatorBuffer.
+- Deployment Funnels (*blue above*):
+    - Contracts that pull funds from the AllocatorBuffer.
+    - The funds can be swapped and/or deployed into AMM pools or specific conduits.
+    - A typical setting for a funnel includes a base rate-limited contract (such as Swapper) and an automation contract on top of it (such as StableSwapper).
+- Conduits (*orange above*):
+    - Yield investment singletons that support deposits and withdrawals.
+
+## Actors
+The allocation system includes several actor types:
+
+- Pause Proxy:
+    - Performs actions through spells with governance delay.
+    - In charge of setting up the core components and the NST minting instant access modules (DC-IAMs).
+    - Ward of the singleton contracts (e.g RWA conduits, Coinbase Custody, AllocatorRoles).
+- AllocatorDAO Proxy:
+    - Performs actions through a sub-spell with governance delay.
+    - Ward of its AllocatorVault and its funnel contracts.
+    - In charge of adding new contracts to the funnel network (e.g Swapper, DepositorUniV3).
+    - Can add operators to its funnel network through the AllocatorRoles contract.
+    - In charge of setting rate-limiting safety parameters for operators.
+- Operator:
+    - Performs actions without a spell and without governance delay.
+    - An optional actor which is whitelisted through the AllocatorRoles contract to perform specified actions on the AllocatorVault, funnels and conduits.
+    - Will typically be a facilitator multisig or an automation contract controlled by one (e.g StableSwapper, StableDepositorUniV3).
+- Keeper:
+    - An optional actor which can be set up to trigger the automation contracts in case repetitive actions are needed (such as swapping NST to USDC every time interval).
+
+![Untitled (1)](https://github.com/makerdao/dss-allocator/assets/130549691/c677928b-32f4-4000-b6ed-e3798caa9c5c)
+
+## Contracts and Configuration
+### VAT Configuration
+
+Each AllocatorDAO has a unique Ilk (collateral type) with one VAT vault set up for it.
+
+- Each ilk supports a 1 trillion NST max debt ceiling.
+- Each Ilk has a special collateral token that is minted and locked in the system.
+- The collateral amount of each vault is 1 million WAD (i.e. 10^6 * 10^18 units of collateral token).
+- All the Ilks have a shared simple [oracle](https://github.com/makerdao/dss-allocator/blob/dev/src/AllocatorOracle.sol) that just returns a fixed price of 1 Million (which multiplied by the collateral amount makes sure the max debt ceiling can indeed be reached).
+
+### AllocatorVault
+
+Single contract per Ilk, which operators can use to:
+
+- Mint (`draw` ) NST from the vault to the AllocatorBuffer.
+- Repay (`wipe`) NST from the AllocatorBuffer.
```

Comment:
Might be worth to mention that you can `draw` and `wipe` to/from other address as well.

Reply:
Resolving this as a new PR is incoming.